wynnset/tapestry-wp
# TYDE: Allow Reordering of Topics In Stage

Closes #513 
Changes:
 - Topic `v-for` in stage is now dependant on `childOrdering` field
 - `childOrdering` is now initialized for all stages 
 - New topic ids are added to stage's `childOrdering` 

Steps to test:
 - Make a stage with 2 children
 - Enter stage edit modal, click Ordering and switch ordering
 - Ensure changes are made on stage screen
